res/values-de/strings.xml

```diff
@@ -41,7 +41,7 @@
 <string name="dialog_backup_done">Datensicherung fertig</string>
 <string name="dialog_backup_failed">Datensicherung fehlgeschlagen!</string>
 <string name="dialog_change_language">Sprache der Seriendetails</string>
-<string name="dialog_use_mirror">TheTVDB Spiegel verwenden (ausprobieren, falls Updates zu lange dauern)</string>
+<string name="dialog_use_mirror">TheTVDB verwenden (falls Updates zu lange dauern)</string>
```

Comment:
It really is a mirror of TheTVDB. Without this enabled, it still uses TheTVDB, kjust the primary server.